share/spice/syntax/syntax.handlebars

```diff
@@ -0,0 +1,3 @@
+
+{{description}}
```

Comment:
I need the handlebar to put a <pre> tag on the code section. Also for whatever reason, if I don't include the description handlebar it doesn't show.